pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/facade/TrainerFacade.java

```diff
@@ -0,0 +1,87 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.facade;
+
+import cz.muni.fi.pa165.pokemon.league.participation.manager.dto.TrainerAuthenticateDTO;
+import cz.muni.fi.pa165.pokemon.league.participation.manager.dto.TrainerChangePasswordDTO;
+import cz.muni.fi.pa165.pokemon.league.participation.manager.dto.TrainerCreateDTO;
+import cz.muni.fi.pa165.pokemon.league.participation.manager.dto.TrainerDTO;
+import cz.muni.fi.pa165.pokemon.league.participation.manager.dto.TrainerRenameDTO;
+import cz.muni.fi.pa165.pokemon.league.participation.manager.dto.PokemonDTO;
+import java.util.List;
+
+/**
+ * Facade interface for object Trainer.
+ *
+ * @author Jiří Medveď 38451
+ */
+public interface TrainerFacade {

```

Comment:
For the whole class - please update javadoc there should always be basic information about the method (missing from changePassword), also the parameters and the return values should be always documented, for example like this
    /**
     * Add a new pokemon to a trainer.
     *
     * @param trainerId Trainer id to add pokemon.
     * @param pokemonId Pokemon id added ti trainer.
     */
    public void addPokemon(Long trainerId, Long pokemonId);